src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Helpers/DelegationHelper.cs

```diff
@@ -52,7 +52,7 @@ public static class DelegationHelper
         public static bool TryGetCoveredByPartyIdFromMatch(List<AttributeMatch> match, out int coveredByPartyId)
         {
             coveredByPartyId = 0;
-            if (match?.Count == 1 && match.First().Id == AltinnXacmlConstants.MatchAttributeIdentifiers.PartyAttribute && int.TryParse(match.First().Value, out coveredByPartyId))
+            if (match?.Count == 1 && match.First().Id == AltinnXacmlConstants.MatchAttributeIdentifiers.PartyAttribute && int.TryParse(match.First().Value, out coveredByPartyId) && coveredByPartyId != 0)
```

Comment:
For readability the int.TryParse could be done in line 54 instead of including it in the if-statement. Same goes for line 70

Reply:
I split those lines in two in an attempt to make them more readable.